### DISCUSSION TITLE
Limit on amount of categories?

### DISCUSSION THREAD

**Discussion by TravisC1** [magicbell/community#57] [Questions] [answered]
Just wondering if there's a hard limit on the amount of categories that can be made per project?

**Comment by unamashana** [comment, marked as answer]
Hey @TravisC1 , as of now there is no limit on the number of categories. How many are you anticipating?

**Reply by TravisC1** [reply]
Our users can be part of multiple organisations so we were considering using categories to divide notifications per organisation, but our conversation has moved on a bit since then and at this stage the plan will be to use custom attributes to capture that and then filter via those instead.

**Reply by unamashana** [reply]
@TravisC1 In that case, I'd suggest using an `external_id` that also includes the `org-id` from your database. This way, you can have separate preferences etc, too.

**Reply by TravisC1** [reply]
We did consider using `{id}_{org_id}` for `external_id` however we also need to send non-org-specific notifications to the user and we don't want to send those to each org account individually as that could be confusing (they read/action it on one account then log into the next and it's unread again).
